src/components/icon/index.tsx

```diff
@@ -53,10 +61,12 @@ const Icon = forwardRef((props: Props, ref: any) => {
     assetName,
     modifiers,
     recorderTag,
+    width,
+    height,
     ...others
   } = props;
   const {margins} = modifiers;
-  const iconSize = size ? {width: size, height: size} : undefined;
+  const iconSize = size ? {width: width ?? size, height: height ?? size} : undefined;
```

Comment:
The props might be missing but, but doesn't they work without this change? The props are spread in both cases, so if you do not send `size` it should work.
* I did not test it.

Reply:
it doesn't work without it because it was spreading iconSize after spreading the props

Reply:
on web size is always defined, that's why we need to give width & height priority.
if size isn't defined (mobile and no one passed size) then width and height will be passed when spreading the props

Reply:
If this is only for the SVG part, I suggest changing this line:
```
const renderSvg = () => <SvgImage fsTagName={recorderTag} data={source} {...props} {...iconSize}/>;
```
to
```
const renderSvg = () => <SvgImage fsTagName={recorderTag} data={source} {...iconSize} {...props}/>;
```

Reply:
This will remove the need for all the other changes